app/services/rollover_reporting_service.rb

```diff
@@ -0,0 +1,71 @@
+class RolloverReportingService
+  def initialize
+    @rollover_scope = RecruitmentCycle.current_recruitment_cycle
+  end
+
+  class << self
+    def call
+      new.call
+    end
+  end
+
+  def call
+    if @rollover_scope.next.present?
```

Comment:
This should be a guard clause

```suggestion
    return {} unless @rollover_scope.next.present?
    
```

Reply:
thanks